### DISCUSSION TITLE
Non privilege user can list all people and assets in his location

### DISCUSSION THREAD

**Discussion by gxlvmesa** [snipe/snipe-it#13398] [General]
Hi, I just started with Snipe-IT, but I have some issue, may be my ignorance, but I want to ask you. After deploy Snipe-IT and do the sync with my Adctive Directory, all people are asigned to same location, so when a user with non privileges make login, in the information screen,  clicking on his location link, in the next screen he is able to list all people and asset assigned to same location including Employed ID for all people in same location. Is it normal? There forbide this? Is it a bug?
this is an example
![image](https://github.com/snipe/snipe-it/assets/135045260/1a36265d-df14-4509-9019-26a6218a094e)

![image](https://github.com/snipe/snipe-it/assets/135045260/ea37d457-622c-4e15-9adb-6c11284a1b21)

**Comment by brianhoganm** [comment]
LOL!! I had to laugh when I saw this. I have known about this for awhile :). I have put in a feature request as well as other have to allow non privilege users to list all assets that belong to their location they manage. (Most people would love to have this (especially in schools) In this situation, when clicking "Print Assigned" it allows you to view assets and people like you mentioned. Either no one has really noticed this or a permission set needs to be created for those two buttons.
